pyproject.toml

```diff
@@ -1,24 +1,76 @@
 [build-system]
-requires = ["setuptools>=61.0"]
+requires = ["setuptools >= 61.0", "setuptools_scm"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools_scm]
+version_scheme = "guess-next-dev"
+local_scheme = "node-and-date"
+
 [project]
-name = "charm-lib-contextual-status"
-version = "0.0.0"
-authors = [
-  {name="George Kraft", email="george.kraft@canonical.com"},
-]
+name = "charms.contextual-status"
```

Comment:
here is the rename from `charm-lib-contextual-status` -> `charms.contextual-status`

Reply:
Hmm. Now that you mention it, this looks like the old reactive naming convention (e.g., `charms.reactive`). What if we use the same convention for "modern" libraries, such as `ops.manifests`

Reply:
we can rename this to `ops.contextual-status`.  I think if we do so, we'll need to make sure that there are no repos/builds trying to use install from this git repo. 

notice [in kubernetes-control-plane](https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/blob/main/src/charm.py#L18)


```python
import charms.contextual_status as status
```

these would need to renamed if they pip installed from this git repo.  I'd hoped instead to just rename this library
`charms.contextual-status` to make it clear the home for this library.